docs/en/ingest-management/data-streams.asciidoc

```diff
@@ -0,0 +1,105 @@
+data–meaning no more indices with hundreds of thousands of fields.
+This leads to better space efficiency and faster queries.
+As an added bonus, only relevant fields are shown in Discover.
+
+* *More granular data control*: For example, filesystem, load, cpu, network, and process metrics are sent
+to different indices–each potentially with its own rollover, retention, and security permissions.
+
+* *Flexible*: Use the custom namespace component to divide and organize data in a way that
+makes sense to your use-case or company.
+
+* *Fewer ingest permissions required*: Data ingestion only requires permissions to append data.
+
+[discrete]
+[[data-streams-naming-scheme]]
+== Data stream naming scheme
+
+{agent} uses the Elastic data stream naming scheme to name data streams.
+The naming scheme splits data into different streams based on the following components:
+
+`type`::
+A generic `type` describing the data, such as `logs`, `metrics`, `traces`, or `synthetics`.
+// Corresponds to the `data_stream.type` field.
+
+`dataset`::
+The `dataset` is defined by the integration and describes the ingested data and its structure for each index.
+For example, you might have a dataset for process metrics with a field describing whether the process is running or not,
+and another dataset for disk I/O metrics with a field describing the number of bytes read.
++
+For APM data, the `dataset` is the instrumented service's `service.name`.
+
+`namespace`::
+A user-configurable arbitrary grouping, such as an environment (`dev`, `prod`, or `qa`),
+a team, or a strategic business unit.
+A `namespace` can be up to 100 bytes in length (multi-byte characters will count toward this limit faster).
+Using a namespace makes it easier to search the data from a given source by using index patterns, or to give users permissions to data by assigning an index pattern to user roles.
+// Corresponds to the `data_stream.dataset` field.
+
+The naming scheme separates each components with a `-` character:
+
+[source,text]
+--
+<type>-<dataset>-<namespace>
+--
+
+For example, if you've set up the Nginx integration with a namespace of `prod`,
+{agent} uses the `logs` type, `nginx.access` dataset, and `prod` namespace to store data in the
+`logs-nginx.access-production` data stream.
+If you use the APM integration with a namespace of `dev`, and `service.name` (dataset) of `frontend`,
+{agent} uses the `traces-apm.frontend.dev` data stream.
+
+All data streams, and the pre-built dashboards that they ship with,
+are viewable on the Fleet Data Streams page:
+
+[role="screenshot"]
+image::images/kibana-fleet-datastreams.png[Data streams page]
+
+TIP: If you're familiar with the concept of indices, you can think of each data stream as a separate index in {es}.
+Under the hood though, things are a bit more complex.
+All of the juicy details are available in {ref}/data-streams.html[{es} Data streams].
```

Comment:
Nice! Knowing "juicy details" is way more intriguing than having to read some other docs. :->